tools/make/ffi.toml

```diff
@@ -126,30 +120,22 @@ category = "ICU4X Development"
 script_runner = "@duckscript"
 script = '''
 exit_on_error true
-cd ffi/diplomat/cpp/examples/pluralrules
-rm -f ../../../../../target/debug/libicu_capi_staticlib_test.a
-exec --fail-on-error make
-cd ../properties
-rm ../../../../../target/debug/libicu_capi_staticlib_test.a
-exec --fail-on-error make
-cd ../bidi
-rm ../../../../../target/debug/libicu_capi_staticlib_test.a
+cd ffi/diplomat/cpp/examples/properties
 exec --fail-on-error make
 cd ../segmenter
-rm ../../../../../target/debug/libicu_capi_staticlib_test.a
```

Comment:
Issue: I think the reason we were rebuilding the .a file was because the feature sets are different. Some have `--features cpp_default` and others have `--features provider_test`, for example.

Reply:
Cargo knows to rebuild when features change. As we now always force the cargo invocation, we don't have to worry about this.

Reply:
ok, and the `FORCE` causes the cargo invocation to always run.